src/python/pants/pantsd/service/scheduler_service.py

```diff
@@ -135,7 +136,7 @@ def _process_event_queue(self):
     try:
       subscription, is_initial_event, files = (event['subscription'],
                                                event['is_fresh_instance'],
```

Comment:
I don't know if others would find this appropriate, but fixing the indentation of this line sounds like a good idea and a small change.